File: 1302-delete-characters-to-make-fancy-string/delete-characters-to-make-fancy-string.cpp
class Solution {
public:
    string makeFancyString(string s) {
        string ans;
        int count = 1;
        ans.push_back(s[0]);
        for(int i = 1;i<s.size();i++){
            if(s[i] == s[i-1]){
                count++;
                if(count > 2) {
                    continue;
                }
                ans.push_back(s[i]);
            } else {
                count = 1;
                ans.push_back(s[i]);
            }
        }
        return ans;
    }
};

File: 1302-delete-characters-to-make-fancy-string/delete-characters-to-make-fancy-string.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        ans = []
        ele = s[0]
        ans.append(ele)
        count = 1

        for i in range(1, len(s)):
            if s[i] == ele:
                count += 1
                if count < 3:
                    ans.append(s[i])
            else:
                ele = s[i]
                count = 1
                ans.append(s[i])
        return "".join(ans)
        